biome.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
we need to add a rule to ignore the .agentuity folder since this is generated code 

https://biomejs.dev/reference/configuration/#filesignore